tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -272,3 +272,37 @@ DRAW_TEST_F(CGContext, ClipBoundingBox, WhiteBackgroundTest<>) {
     CGContextSetRGBFillColor(context, 0.0, 1.0, 0.0, 1.0);
     CGContextFillRect(context, CGContextGetClipBoundingBox(context));
 }
+
+DRAW_TEST_F(CGContext, SubPixelAlignment, WhiteBackgroundTest<>) {
```

Comment:
this test should almost certainly be drawing _at_ the half pixel mark, not transforming the world to that the world aligns at the half pixel mark.

also, it should just use CGContextStrokeRect.